tests/logging/CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
 include(EkatCreateUnitTest)
 
 # Test basic logger capabilities
-EkatCreateUnitTest(serial_file_log serial_file_log_tests.cpp LIBS ekat)
+EkatCreateUnitTest(serial_file_log serial_file_log_tests.cpp
+  LIBS Logging)
 
 EkatCreateUnitTest(mpi_file_log_tests mpi_file_log_tests.cpp
-  LIBS ekat
+  LIBS Logging
```

Comment:
I probably missed this: are these library targets namespaced under `ekat::` or something in CMake so these names have a discernable context?

Reply:
I did add a namespace when I installed the targets. But I haven't tested it yet. Besides, that would only work if ekat is _installed_, rather than added via `add_subdirectory`. I don't know if I can decl libs directly with a namespace, like `add_library(ekat::Core ...)`. Maybe? I'll try. If it works, I'll just switch to that. Or would you prefer the libs to just be called `EkatCore`, `EkatKokkosUtils`,...?